### ISSUE TITLE
PIL setfont() deprecated

### ISSUE THREAD

**Issue by methodds** [amueller/word_cloud#96]
Hi,

when using word_cloud on windows with python 3.5 (Anaconda), plotting a text works perfectly fine but I always receive a warning from Anaconda refering to PIL: 
"C:\Anaconda3\lib\site-packages\PIL\ImageDraw.py:104: UserWarning: setfont() is deprecated. Please set the attribute directly instead.
  "Please set the attribute directly instead.")"

Is there another way to set the font without receiving the warning? This is the code I used:

```python
wordcloud = WordCloud( font_path='C:\Windows\Fonts\calibrib.ttf', max_font_size=40,
                      width = 800, height=600, stopwords=stopwords_eng).generate(tweet_txts)
```

Thanks for you work!

**Comment by amueller** [comment]
well they say to set the attribute directly ;) PR welcome.

**Comment by amueller** [comment]
fixed in #106
